### ISSUE TITLE
Add metadata support

### ISSUE THREAD

**Issue by jmfee-usgs** [usgs/hazdev-geoserve-ws#86]
To request supported regions/places endpoint types.  Use JSON output format.

**Comment by jmfee-usgs** [comment]
Add a method `metadata` to index.php, GeoserveWebService.

Should output a list of available `endpoint`s, one for `places` and one for `regions`.
```
{
  endpoints: ['places', 'regions'],
  places: {
    types: []
  },
  regions: {
    types: []
  }
}
```

**Comment by emartinez-usgs** [comment]
Base URL: /ws/geoserve/
 - /index.html → HTML formatted output documenting the available endpoints in the web service.
 - /index.json → JSON formatted output documenting the available endpoints in the web service.


 - /places.html → HTML formatted output documenting places endpoint including available layers within this endpoint.
 - /places.json → JSON formatted output documenting places endpoint including available layers within this endpoint.
 - /places.json?parameters → JSON formatted output with places endpoint results or a JSON formatted error message


 - /regions.html → HTML formatted output documenting regions endpoint including available layers within this endpoint.
 - /regions.json → JSON formatted output documenting regions endpoint including available layers within this endpoint.
 - /regions.json?parameters → JSON formatted output with regions endpoint results or a JSON formatted error message.

Notes
 - Proper HTTP headers should be used throughout
 - Map each endpoint (metadata/places/regions) to a file on disk using rewrites

**Comment by emartinez-usgs** [comment]
See pull request #114.

**Comment by jmfee-usgs** [comment]
Fixed by #114
